lib/actions/purgeTable.js

```diff
@@ -0,0 +1,77 @@
+let tableName = null;
+let dynamodb = null;
```

Comment:
Really confusing way of using those - declaring variables and then using `this` from bound function to access them.

Please just not store them as globals and pass through the function calls instead.